src/Components/Error/Error.js

```diff
@@ -6,10 +7,14 @@ const Error = ({ error }) => {
     <div className="error-container">
       <img className="error-image" src={noVoice1}
         alt="microphone with horizontal strikethrough" />
-      <h3 className="error-text">We're sorry, please try again.</h3>
+      <h3 className="error-text">
+        <FormattedMessage
+          id="error.sorryMessage"
+          defaultMessage="We're sorry, please try again." />
+      </h3>
```

Comment:
As of the time that I'm doing this code review, this is one place where a test is failing.